ui/webui/test/check-storage

```diff
@@ -866,5 +866,33 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         r.check_disk_row(disk, 3, f"{vgname}-home, 8.12 GB: mount, /home")
         r.check_disk_row(disk, 4, f"{vgname}-swap, 902 MB: mount, swap")
 
+    @nondestructive
+    def testUnusableFormats(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        s = Storage(b, m)
+
+        self.addCleanup(m.execute, "wipefs --all /dev/vda")
+
+        disk = "/dev/vda"
+        dev = "vda"
+        s.partition_disk(disk, [("1GB", "exfat")])
+        s.udevadm_settle()
+
+        i.open()
+        i.next()
+        s.rescan_disks()
+
+        self.select_mountpoint(i, s, [(dev, True)])
+
+        # exfat filesystem cannot be reformatted by blivet
```

Comment:
Unfortunately exFAT is currently broken in Fedora which causes the test to fail:

```
testlib.Error: FAIL: Test completed, but found unexpected journal messages:
Process 7872 (tune.exfat) of user 0 dumped core.
```

I'll try to find a different way to test this.

Reply:
Switched to UDF for the test case. Lets hope it won't cause any issues.